CONTRIBUTING.rst

```diff
@@ -264,6 +284,19 @@ If you wish to test a specific branch using GitHub CI, this can be set in `.gith
     In order for a Pull Request to be allowed to merge to main development branch, this variable must match the latest tagged commit name on `Ouranosinc/xclim-testdata`.
     We suggest merging changed testing data first, tagging a new version of `xclim-testdata`, then re-running tests on your Pull Request at `Ouranosinc/xclim` with the newest tag.
 
+Running Tests Offline
+~~~~~~~~~~~~~~~~~~~~~
+
+`xclim` testing is normally performed with the assumption that the machine running the tests has internet access. Many calls to `xclim` functions will attempt to download data or verify checksums from the `Ouranosinc/xclim-testdata` repository. This can be problematic for developers working on features when internet access is not immediately available.
+
+In order to bypass these remote validation checks, the testing can be run offline by either running pytest with the following options::
+
+    $ pytest --disable-socket --allow-unix-socket -m "not requires_internet"
```

Comment:
Is there a way to have a simple offline shortcut for pytest too ?

Reply:
Calling `pytest` directly here would involve some really wacky workarounds. Would you be opposed to having a Makefile recipe (something resembling `make test-offline`)  to cover this case?

Reply:
The downside to the Makefile approach would be that you wouldn't benefit from `pytest` IDE integrations (not sure if others use them). There really isn't a way to win here.

Reply:
Let`s leave it as is then.